BMP388_I2C/BMP388_I2C.sch

```diff
@@ -3,7 +3,7 @@ EELAYER 30 0
 EELAYER END
```

Comment:
Switch to star-common-lib:BMP388. Right now is on some weird core-rescue version.